include/ignition/rendering/LidarVisual.hh

```diff
@@ -171,6 +171,15 @@ namespace ignition
       /// \return The type for lidar visual
       public: virtual LidarVisualType Type() const = 0;
 
+      /// \brief Set size of the lidar visualization, e.g. size of lidar points
```

Comment:
Is there a unit for this? Is it "pixels" even though it's a double?

Reply:
updated doc. 43c4328 It's in pixels. It eventually comes down to setting `gl_PointSize` which is a float and I saw some examples setting it to a floating point value so I left it here as a double.